toolkit/tools/internal/grub/commands.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package grub
```

Comment:
Overall comment on directory structure
Instead of having these libraries in internal/, we can have it in pkg/. Since both internal/ and package are rooted at tools/, they essentially have the same "visibility" in our codebase.

Reply:
The visibility is the same within our codebase, yes. But if a package from outside our codebase uses this code, then they'll be able to access stuff under `pkg` but not `internal`. So, I think it makes sense to limit `pkg` to only the library equivalents of the toolkit's tools.